frontend/components/buttons/Button/Button.tsx

```diff
@@ -57,9 +50,8 @@ interface Inputs {
 
 class Button extends React.Component<IButtonProps, IButtonState> {
   static defaultProps = {
-    size: "",
     type: "button",
-    variant: "brand",
+    variant: "default",
```

Comment:
`"brand"` renamed to `"default"` and removed all the explicitly set `variant` throughout the code